README.md

```diff
@@ -62,7 +72,8 @@ compiled_model = core.compile_model(ov_model, 'CPU')
 output = compiled_model({0: example.numpy()})
 ```
 
-**TensorFlow Model**
+<details>
```

Comment:
what is a reason to hide? I think these two chunks of code for TF and PT are the most important in this README to which developers will take a look

Reply:
yeah agree
I just tested some options to save on space if we ever want to add more/llms examples, added back